config/master-machine.yaml

```diff
@@ -12,7 +12,7 @@ spec:
       node-role.kubernetes.io/master: ""
   providerSpec:
     value:
-      apiVersion: alibabacloudproviderconfig.openshift.io/v1alpha1
+      apiVersion: alibabacloudproviderconfig.openshift.io/v1
```

Comment:
i'm pretty sure these should be `machine.openshift.io/v1` too

Reply:
i was wrong, looking at aws example here https://docs.openshift.com/container-platform/4.9/machine_management/creating_machinesets/creating-machineset-aws.html